pkgs/by-name/go/goldwarden/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, makeWrapper
+, libfido2
+, dbus
+, pinentry
+}:
+
+buildGoModule rec {
+  pname = "goldwarden";
+  version = "0.2.7";
+
+  src = fetchFromGitHub {
+    owner = "quexten";
+    repo = "goldwarden";
+    rev = "v${version}";
+    hash = "sha256-OXJovoJ2+YIMqzoG6J2LlxUC5DMZRAdEl+ZEv6PDXlI=";
+  };
+
+  vendorHash = "sha256-1Px60+f23qoP5eEOUC3WG5vKJYjbD3bPOrDyBpXlMT0=";
+
+  ldflags = [ "-s" "-w" ];
+
+  nativeBuildInputs = [makeWrapper];
+
+  buildInputs = [libfido2];
+
+  postInstall = ''
+    wrapProgram $out/bin/goldwarden \
+      --suffix PATH: ${lib.makeBinPath [dbus pinentry]}
+
+    mkdir -p $out/usr/share/polkit-1/actions/
+    install -Dm644 $src/resources/com.quexten.goldwarden.policy $out/usr/share/polkit-1/actions/com.quexten.goldwarden.policy
+  '';
+
```

Comment:
Can we add a `passthru.updateScript` here? Will make it much easier to update. `nix-update-script` is usually a solid option

Reply:
Done.